README.md

```diff
@@ -1,58 +1,50 @@
-> [!NOTE]
-> Version 5.0 is currently in release candidate period. Bug bounty rewards are boosted 50% until the release.  
-> [See more details on Immunefi.](https://immunefi.com/bounty/openzeppelin/)
-
 # <img src="logo.svg" alt="OpenZeppelin" height="40px">
 
+*The most widely adopted library for secure smart contract development in Solidity.*
```

Comment:
I like having a tagline here but not sure if "most adopted" is what we'd like to be known for.

```suggestion
*The most secure and widely adopted library for smart contract development in Solidity.*
```

Reply:
I tend to prefer zack's formulation. Its undeniable that we are the most widely adopted ... but the most secure is possibly up to debate. We don't want to involve the legal team here :P

Reply:
It'll be deniable is the most widely adopted if we solely focus on promoting that. I'd strongly prefer to highlight security over popularity.